tests/ui/impl-trait/precise-capturing/bad-lifetimes.rs

```diff
@@ -1,14 +1,14 @@
 #![feature(precise_capturing)]
 //~^ WARN the feature `precise_capturing` is incomplete
 
-fn no_elided_lt() -> impl use<'_> Sized {}
+fn no_elided_lt() -> impl Sized + use<'_> {}
```

Comment:
why are all these tests changed?

Reply:
is this the new idiomatic order of things?

Reply:
yeah -- that's the style that T-lang was suggesting as idiomatic